src/makeCalculator.js

```diff
@@ -5,6 +5,41 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+    add(x) {
+      this.result += x;
+
+      return this;
+    },
+    subtract(x) {
+      this.result -= x;
+
+      return this;
+    },
+    multiply(x) {
+      this.result *= x;
+
+      return this;
+    },
+    divide(x) {
+      this.result /= x;
+
+      return this;
+    },
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+    operate: function (callbak, x) {
+      callbak.call(this, x);
+
+      return this;
+    },
```

Comment:
The `operate` method is using the `function` keyword, which is unnecessary here. You can use the method shorthand to define this function, which is more concise and is the preferred style in modern JavaScript.